test/jdk/java/io/InputStream/NullInputStream.java

```diff
@@ -21,13 +21,15 @@
  * questions.
  */
 
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
 import java.io.ByteArrayOutputStream;
 import java.io.EOFException;
-import java.io.InputStream;
 import java.io.IOException;
-import org.testng.annotations.AfterGroups;
-import org.testng.annotations.BeforeGroups;
-import org.testng.annotations.Test;
+import java.io.InputStream;
+
 import static org.testng.Assert.*;
```

Comment:
Would it not be more standard to put the new imports just after this import? Same comment applies in the other files.

Reply:
That's IntelliJ magic.  I can update if you prefer and can let IntelliJ optimize all of the imports

Reply:
Ah. I was just thinking of consistency with other tests. Up to you.

Reply:
I think your milage will vary depending on the author and the IDE being used.  My primary concern was to address the issue for the failing test and Intellij arranged the imports as they are above.  I guess I am less concerned about the imports as I think in some cases it comes to personal preference.

So I would prefer to leave as is if you are OK :-)

Reply:
I think that's fine.